orchagent/portsorch.cpp

```diff
@@ -1014,10 +1049,20 @@ bool PortsOrch::setPortAdminStatus(Port &port, bool state)
         }
     }
 
-    SWSS_LOG_INFO("Set admin status %s to port pid:%" PRIx64,
-                    state ? "UP" : "DOWN", port.m_port_id);
-
-    setGearboxPortsAttr(port, SAI_PORT_ATTR_ADMIN_STATE, &state);
+    bool gbstatus = setGearboxPortsAttr(port, SAI_PORT_ATTR_ADMIN_STATE, &state);
+    if (gbstatus != true)
+    {
+        SWSS_LOG_ERROR("Failed to set admin status on PHY %s to port pid:%" PRIx64,
+                       state ? "UP" : "DOWN", port.m_port_id);
+     /* add task handler to recover/take action on this failure */
```

Comment:
please indent the comment 4 spaces to the right.
Also, this is something to be done. Then, should this be mentioned as TBD or appropriate marker?